pkg/gator/reader/read_constraints.go

```diff
@@ -177,6 +177,20 @@ func ReadConstraint(f fs.FS, path string) (*unstructured.Unstructured, error) {
 	return u, nil
 }
 
+func ReadExpansion(f fs.FS, path string) (*unstructured.Unstructured, error) {
+	u, err := ReadObject(f, path)
+	if err != nil {
+		return nil, err
+	}
+
+	gvk := u.GroupVersionKind()
+	if gvk.Group != "expansion.gatekeeper.sh" {
```

Comment:
We should test both group and kind here (we don't do that for constraints because constraints have an unknown set of kinds that all live under the same group)